bitnami/kube-prometheus/values.yaml

```
@@ -3308,6 +3308,10 @@ rbac:
   create: true
   pspEnabled: true
 
+## @param extraRbac Extra Rbac for the Prometheus clusterrole
+##
+extraRbac: []
```

We do support this in other charts via the `rbac.rules` parameter, let's stick to the format we used on them for consistency:

```yaml
  ## @param rbac.rules Custom RBAC rules to set on Prometheus ClusterRole
  ## e.g:
  ## rules:
  ##   - apiGroups:
  ##       - ""
  ##     resources:
  ##       - pods
  ##     verbs:
  ##       - get
  ##       - list
  ##
  rules: []
```

---

That said and, given that we have 2 ClusterRole(s) (one for prometheus and another for prometheus-operator):

- https://github.com/bitnami/charts/blob/main/bitnami/kube-prometheus/templates/prometheus/clusterrole.yaml
- https://github.com/bitnami/charts/blob/main/bitnami/kube-prometheus/templates/prometheus-operator/clusterrole.yaml

We might need to split this parameter in two so both can be customize separately:

```suggestion
  ## @param rbac.rules.operator Custom RBAC rules to set on Prometheus Operator ClusterRole
  ## @param rbac.rules.prometheus Custom RBAC rules to set on Prometheus ClusterRole
  ## e.g:
  ## rules:
  ##   operator: []
  ##   prometheus:
  ##     - apiGroups:
  ##         - ""
  ##       resources:
  ##         - pods
  ##       verbs:
  ##         - get
  ##         - list
  ##
  rules:
    operator: []
    prometheus: []
```

---

Thanks, I'm on holidays for two weeks, please keep it opens I will fix that when I come back
Regards